src/components/overview.jsx

```diff
@@ -10,19 +10,19 @@ class Overview extends React.Component {
   findPlayground(className) {
     return ReactDOM.findDOMNode(this.refs.overview).getElementsByClassName(className);
   }
+  mountContainer(noRender, source) {
```

Comment:
prefer param order to go from less specific to more specific. Also, then when this option is false you can just leave it off.